pkg/setting/setting.go

```diff
@@ -449,6 +449,10 @@ func (cfg Cfg) IsDatabaseMetricsEnabled() bool {
 	return cfg.FeatureToggles["database_metrics"]
 }
 
+func (cfg Cfg) IsSwaggerUIEnabled() bool {
+	return cfg.FeatureToggles["swagger-ui"]
```

Comment:
:thinking: -- I expect this will need to use the newly added "featuremgmt" class to define and check flags.

1. Add a flag to:  https://github.com/grafana/grafana/blob/main/pkg/services/featuremgmt/registry.go#L5
2. run the test: https://github.com/grafana/grafana/blob/main/pkg/services/featuremgmt/toggles_gen_test.go#L17
3. check the feature flag using the defined constants